Tests/scripts/hook_validations/image.py

```diff
@@ -20,10 +20,18 @@ def __init__(self, file_path):
             self.file_path = file_path
         else:
             if re.match(INTEGRATION_YML_REGEX, file_path, re.IGNORECASE):
-                self.file_path = glob.glob(os.path.join(os.path.dirname(file_path), '*.png'))[0]
+                try:
+                    self.file_path = glob.glob(os.path.join(os.path.dirname(file_path), '*.png'))[0]
+                except IndexError:
+                    self._is_valid = False
+                    print("You've created/modified a package but failed to provide an image as a .png file, "
```

Comment:
use print_error for all errors
```suggestion
                    print_error("You've created/modified a package but failed to provide an image as a .png file, "
```